packages/react/src/components/Forms/Select/index.tsx

```diff
@@ -1,16 +1,23 @@
-import React from "react";
+import React, { forwardRef } from "react";
 import { SelectMulti } from ":/components/Forms/Select/multi";
 import { SelectMono, SelectProps } from ":/components/Forms/Select/mono";
 
 export * from ":/components/Forms/Select/mono";
 export * from ":/components/Forms/Select/multi";
 
-export const Select = (props: SelectProps) => {
+export interface SelectHandle {
+  blur: () => void;
+}
+export const Select = forwardRef<SelectHandle, SelectProps>((props, ref) => {
```

Comment:
`SelectMono` already handle definition of `SelectProps` and `Option`.
we could continue to use it as the container for other types/interfaces